Tests/Utils/ModuleDependancy.inl

```diff
@@ -1,16 +1,28 @@
 #pragma once
 
-template <unsigned int APlatform, class AAutoModule> ModuleDependancy<APlatform, AAutoModule>::ModuleDependancy(AAutoModule& AReferenceToManagerObject) :
-	FReferenceToManagerObject(AReferenceToManagerObject)
+template <typename AAutoModule> struct RefHookStruc {
+	void* FAM;
+};
+
+template <typename AAutoModule> void CYB::Platform::Modules::Manager::Backdoor(AAutoModule& AHooker) {
+	reinterpret_cast<RefHookStruc<AAutoModule>&>(AHooker).FAM = CYB::Core().FModuleManager.GetAutoModule<AAutoModule>();
```

Comment:
I love ENABLE_TEST_HOOKS. No one can even say this is illegal, because it isn't ;)